components/nested_star/.cargo/config.toml

```diff
@@ -0,0 +1,3 @@
+[net]
+git-fetch-with-cli = true
```

Comment:
Does this belong in the repo?

Reply:
I think this can be removed now. This was added when the nested-sta-rs repo was private so that the git executable can handle authentication.